rust/candid_parser/src/bindings/javascript.rs

```diff
@@ -237,38 +245,84 @@ fn pp_actor<'a>(ty: &'a Type, recs: &'a BTreeSet<&'a str>) -> RcDoc<'a> {
     }
 }
 
+fn pp_deprecation_comment<'a>() -> RcDoc<'a> {
+    str("/**").append(RcDoc::hardline())
+        .append(" * @deprecated Import IDL types directly from this module instead of using this factory function.")
```

Comment:
We should also add a note that this is possible only on @dfinity/candid v3.0.1, something like:

```suggestion
        .append(" * @deprecated Since `@dfinity/candid` v3.0.1, you can import IDL types directly from this module instead of using this factory function.")
```

Reply:
Done